scan/freebsd.go

```diff
@@ -176,14 +176,18 @@ func (o *bsd) scanInstalledPackages() (models.Packages, error) {
 }
 
 func (o *bsd) scanUnsecurePackages() (models.VulnInfos, error) {
-	const vulndbPath = "/tmp/vuln.db"
-	cmd := "rm -f " + vulndbPath
+
+	vulndbPath := "/tmp/vuln.db"
+	cmd := util.PrependProxyEnv("pkg audit -F -r -f " + vulndbPath)
+	if o.ServerInfo.PkgAuditPath != "" {
+		vulndbPath = o.ServerInfo.PkgAuditPath
+		cmd = util.PrependProxyEnv("pkg audit -f " + vulndbPath)
```

Comment:
Why did you remove -r option?
https://www.freebsd.org/cgi/man.cgi?query=pkg-audit&sektion=8&n=1
```
     -r, --recursive
	    Prints packages that depend	on vulnerable packages and are thus
	    potentially	vulnerable as well.
```

Reply:
My apologies will fix and re submit today.